File: Day13/perfectSquareChecker.js
// Function to check if a number is a perfect square
function isPerfectSquare(num){
    if(num <0 ){
        return false; // Negative number can not be a perfect square
    }
    let i =0;
    while(i*i <= num){
        if(i*i == num){
            return true;
        }
        i++;
    }
    return false;
}
num = parseInt(prompt("Enter the number:"));

//Store the result in a variable and display it
let ans = isPerfectSquare(num);
if(ans){
    console.log(num +" is a perfect square.");
}
else{
    console.log(num +" is a perfect square.");
}

File: Day13/perfect_square_checker.py

""" Function to check if a number is a perfect square """

def is_perfect_square(num):
    """Negative number can not be a perfect square"""
    if num <0 :
        return False

    i =0
    while i*i <= num:
        if i*i == num:
            return True
        i = i + 1
    return False

number = int(input("Enter the number: "))

#Store the result in a variable and display it

ANS = is_perfect_square(number)

if ANS:
    print(number , " is a perfect square.")

else:
    print(number , " is not a perfect square.")
